### PR TITLE
Per other commands, make progress dots in req only w/ -verbose

### PR DESCRIPTION
**Pull request by pprindeville**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by t8m** [comment]
AFAIK for the keygen commands we did not change the default - i.e. by default the progress is still reported there.

**Comment by t8m** [comment]
But I do not think we want to change the default for verbose to be 1 here as that has further implications. IMO this should use some additional variable such as `progress` - and that would be 1 by default, set to 0 by quiet and set to 1 by verbose.

File: doc/man1/openssl-req.pod.in [hunk 1]
```diff
@@ -402,6 +403,11 @@ Non-interactive mode.
 
 Print extra details about the operations being performed.
 
+=item B<-quiet>
+
+Print fewer details about the operations being performed, which may be
+handy during batch scripts or pipelines.
```
Comment (t8m) [inline doc/man1/openssl-req.pod.in:409]: Maybe explicitly mention that keygen progress is disabled?

Comment (pprindeville) [reply doc/man1/openssl-req.pod.in]: Done

**Review by t8m — APPROVED** [review]

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by beldmit** [comment]
The test failure seems irrelevant

**Comment by paulidale** [comment]
> But I do not think we want to change the default for verbose to be 1 here as that has further implications. IMO this should use some additional variable such as `progress` - and that would be 1 by default, set to 0 by quiet and set to 1 by verbose.

We really should make an effort to get consistency across the commands.  Possibly for version 4.

**Review by paulidale — APPROVED** [review]

**Comment by pprindeville** [comment]
I think that's the last of the progress-emitting commands... but yeah, it wouldn't be bad to make progress dots turned off by default for 4.0.

**Comment by paulidale** [comment]
Merged, thanks for the contribution.

**Pull request closed without merging**